webui/react/src/services/apiBuilder.ts

```diff
@@ -1,8 +1,8 @@
 import axios, { AxiosResponse, CancelToken, Method } from 'axios';
+import * as DetSwagger from 'services/api-ts-sdk';
```

Comment:
this is a local import but is bundled with external package on services/api.ts it's bundled in another way :thinking:

Reply:
eslint would put is as `external` if it doesn't find the local files and would put it next to internal ones if it does.

Reply:
![Screen Capture_termite_20200709144045](https://user-images.githubusercontent.com/12127420/87093528-2e39ae80-c1f2-11ea-8e98-0bc4889564c9.png)

Reply:
if we consider services/api-ts-sdk directory as a dependency and get it at get-deps we wouldn't have this issue but there might be more aspects to consider.